pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -8194,11 +8194,19 @@ table_snapshot_opt:
             Expr: $4,
         }
     }
-|   '{' SNAPSHOT '=' expression '}'
+|   '{' SNAPSHOT '=' ident '}'
```

Comment:
what does 'ident' mean?

Reply:
The string type does not require a comma